app/src/components/widgets/charts/investigatedTrendChart/statusPageModeConfig.jsx

```diff
@@ -0,0 +1,142 @@
+import ReactDOMServer from 'react-dom/server';
+import { getItemColor } from 'components/widgets/charts/common/utils';
+import { TimelineTooltip } from './timelineTooltip';
+import { MESSAGES } from './common/constants';
+
+export const mapMinListLength = {
+  '1M': 30,
+  '3M': 12,
+  '6M': 24,
+};
+
+const renderTooltip = (itemData, intl) => (d, defaultTitleFormat, defaultValueFormat, color) => {
```

Comment:
Seems that it is duplicates with `renderTooltip` in the  `../timelineConfig`.
It will be better to create a common method for this two modes